waku/v2/node/rpc_api/waku_api_tools.nim

```diff
@@ -0,0 +1,21 @@
+import
+  waku_api_types,
+  options,
```

Comment:
`std/options` - std for anything from std lib

Reply:
Addressed in 6bea1b3.